webcompat/templates/home-page/form.html

```diff
@@ -56,7 +56,7 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
               {% endif %}
             </div>
             <span class="wc-Form-validation">
-            {{ form.description(class_='wc-Form-item required',
+            {{ form.description(class_='wc-Form-item required mousetrap js-desc',
```

Comment:
One tiny thing. Would you be able to change `js-desc` to `js-Desc` as this is our naming convention? Thanks!

Reply:
sure, that works.